net/peers/peer.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package peers contains helpers for looking up peers on a Tailscale network.
+package peers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+	"os/exec"
+)
+
+// A Peer is another machine connected on a users network.
+type Peer struct {
+	Hostname string `json:"Hostname"`
```

Comment:
This is a little misleading because we're going to be supporting domain-unique host names later (there's a field returned now that's always blank). What this actually is is the non-unique client-reported hostname. I don't think we want to start supporting multiple types with slightly different field sets/names.

Why not just return the tailcfg types?